compiler/src/test/kotlin/batchtest.kt

```diff
@@ -0,0 +1,52 @@
+package dk.aau.cs.d409f19
+
+import dk.aau.cs.d409f19.Utilities.Companion.compileProgram
+import dk.aau.cs.d409f19.cellumata.ErrorLogger
+import dk.aau.cs.d409f19.cellumata.TerminatedCompilationException
+import org.junit.jupiter.api.Test
+import java.io.File
+
+private const val batchDir = "src/main/resources/compiling-programs/"
+
+class BatchTest {
+
+    /**
+     * Tests whether all programs under a given path compiles
+     */
+    @Test
+    fun batchPass() {
+        batchDir.getPrograms().forEach {
```

Comment:
As I have metioned in another comment you should use parameterized tests instead of having a loop in the test

Reply:
Fixed in: 7edc9ae, resolve if sufficient.